api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -453,6 +453,40 @@ message HttpConnectionManager {
   // with `prefix` match set to `/dir`. Defaults to `false`. Note that slash merging is not part of
   // `HTTP spec <https://tools.ietf.org/html/rfc3986>` and is provided for convenience.
   bool merge_slashes = 33;
+
+  message LocalReplyConfig {
+    // Configuration of list of mappers which allows to filter and change local response. 
+    // Code iterate through mappers until first match of filter.
+    repeated ResponseMapper mapper = 1;
+
+    // Allows to define custom format of local reply.
+    oneof reply_format {
```

Comment:
I think something like this belongs in `envoy.type`, as it comes up in other places, e.g. access logs. It's basically a `StringOrJson` type.

Reply:
I couldn't find this type. Maybe it should be added in v3?

Reply:
I'm suggesting we start the trend by adding it in this PR :) Up to you, at least leave a `[#next-major-version:...]` comment if not.

Reply:
Added new format. You can check if it looks ok. I haven't changed it in v2 accesslog config because it isn't compatible with current acceslog format. Example:
```
format:
  string_format: 
```